### PR TITLE
#205: Resource packetfabric_port_loa

### LINKED ISSUE #205: resource packetfabric_port_loa
Generate LOA from port with port_circuit_id as user input

resource `packetfabric_port_loa`
POST [/v2/ports/{port_circuit_id}/letter-of-authorization ](https://docs.packetfabric.com/api/v2/redoc/#tag/Ports/operation/get_port_device_info)

Checklist:
- [ ] tested locally
- [ ] added/updated acceptance tests
- [ ] add examples (in example folder)
- [ ] generate docs

### PR DESCRIPTION
**Pull request by pontovinte**
Signed-off-by: Yrineu Rodrigues <yrineu@pontovinte.com>

## Description

Resource packetfabric_port_loa

Closes #205 

## Checklist

- [x] tested locally
- [x] added/updated acceptance tests
- [x] add examples (in example folder)
- [x] generate docs

### CODE REVIEW

**Comment by medzin** [comment]
I am working on the acceptance test.

**Comment by medzin** [comment]
Tests added.

**Review by elXaptation — DISMISSED** [review]

File: examples/resources/packetfabric_port_loa/resource.tf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+resource "packetfabric_port_loa" "new" {
+  provider = packetfabric
+  port_circuit_id = var.pf_market_port_circuit_id
+  loa_customer_name = "MyName"
+  destination_email = "email@mydomain.com"
+}
```
Comment (rjouhann) [inline examples/resources/packetfabric_port_loa/resource.tf:6]: @elXaptation should run `terraform fmt` on the examples too.

**Review by rjouhann — CHANGES_REQUESTED** [review]

**Review by caitlinwheeless — APPROVED** [review]
No extra doc changes needed here. There were only 3 descriptions from the schema.

**Review by rjouhann — APPROVED** [review]

**Review by caitlinwheeless — APPROVED** [review]
blurb added, now it's good to go

**Pull request merged**